multimedia/welle.io/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               qt5 1.0
+PortGroup               github 1.0
+PortGroup               cmake 1.0
+
+#PortGroup               compilers 1.0
+
+github.setup            AlbrechtL welle.io 2.0-beta3 v
+github.tarball_from     archive
+name                    welle.io
+categories              multimedia
+maintainers             nomaintainer
+
+description             DAB/DAB+ Software Radio
+long_description        This is an open source DAB and DAB+ software defined radio (SDR)  \
+                        with support for rtl-sdr (RTL2832U) and airspy. It supports  \
+                        high DPI and touch displays and it runs even on cheap computers  \
+                        like Raspberry Pi 2/3 and 100€ China Windows 10 tablets.
+
+platforms               darwin
+license                 GPL-2
+license_noconflict      openssl
+
+homepage                https://www.welle.io/
+
+checksums               rmd160  05113c73dc7b97fed256d91cd6a971dabc30c1ce \
+                        sha256  99acaea8386b092ebc80cd6c689ab0106a9f74ad121c04d0ae893f4e35a5a3e0 \
+                        size    1613717
+
+depends_lib-append      port:faad2 \
+                        port:fftw-3-single \
+                        port:lame \
+                        port:libusb \
+                        port:mpg123 \
+                        port:${qt5.name}-qtbase \
+                        port:${qt5.name}-qtcharts \
+                        port:${qt5.name}-qtdeclarative \
+                        port:${qt5.name}-qtmultimedia \
+                        port:${qt5.name}-qtquickcontrols \
+                        port:${qt5.name}-qtquickcontrols2
```

Comment:
This is not the correct way to use the qt5 portgroup. You need to use the `qt5.depends_component` function instead. See other qt5 based Portfiles for reference including qbs and octave.

Reply:
done